src/lib/Scenes/Favorites/index.tsx

```diff
@@ -32,15 +32,24 @@ const Title = styled.Text`
 
 const isStaging = gravityURL.includes("staging")
 
+const WorksTab = 0
+const ArtistsTab = 1
+const CategoriesTab = 2
+
+interface Props {
+  tracking: any
+}
+
 @screenTrack({
   context_screen: Schema.PageNames.SavesAndFollows,
   context_screen_owner_type: null,
 })
-class Favorites extends React.Component<null> {
+class Favorites extends React.Component<Props, null> {
   render() {
     return (
       <View style={{ flex: 1 }}>
         <ScrollableTabView
+          onChangeTab={selectedTab => this.fireTabSelectionAnalytics(selectedTab)}
```

Comment:
JS tip: This is effectively the same as just `onChangeTab={this.fireTabSelectionAnalytics}`, especially since `fireTabSelectionAnalytics` is defined using ES7 class properties.